staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -82,20 +79,18 @@ var _ utilnet.Dialer = &SpdyRoundTripper{}
 
 // NewRoundTripper creates a new SpdyRoundTripper that will use the specified
 // tlsConfig.
-func NewRoundTripper(tlsConfig *tls.Config, followRedirects, requireSameHostRedirects bool) *SpdyRoundTripper {
+func NewRoundTripper(tlsConfig *tls.Config, requireSameHostRedirects bool) *SpdyRoundTripper {
```

Comment:
This is a library function. Let's keep the function signature unless necessary.

Also perhaps the follwoRedirects is still useful in other use cases?

Reply:
updated